src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/common.kt

```diff
@@ -474,6 +480,48 @@ fun <T> runPipelineWithDeadLetterQueue(
     }
 }
 
+fun <T> runTracedPipelineWithDeadLetterQueue(
```

Comment:
Can we remove `runPipelineWithDeadLetterQueue` function?

Reply:
Not until all consumers integrate tracing propagation, ie PR #118